tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -0,0 +1,662 @@
+    \end{equation*}
+\end{multicols}
+
+
+\begin{equation*}
+    \tilde{\pi}_{(0,0)} = (\lambda^A)^2 \mu^4 + 2 \lambda^A \mu^5 + \mu^6
+\end{equation*}
+
+\newpage
+\subsubsection{Conjecture of adding rows}
+
+Let us consider three Markov models with the same number of servers \(C=1\), the same threshold \(T=1\), the same system capacity \(N=2\) but different parking capacity \(M=1\), \(M=2\) and \(M=3\).
+
+
+\begin{multicols}{3}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1121/main.tex}}
+        \caption{\(M=1\)}
+    \end{figure}
+    \columnbreak
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1122/main.tex}}
+        \caption{\(M=2\)}
+    \end{figure}
+    \begin{figure}[H]
+        \centering
+        \scalebox{0.8}{\input{MarkovChain/closed_form_state_probs/example_model_1123/main.tex}}
+        \caption{\(M=3\)}
+    \end{figure}
+\end{multicols}
+
+By increasing the parking capacity of the model it can be easily observed that the number of spanning trees rooted at \((0,0)\) increases as well since more combinations of paths can be generated using the new edges and vertices. The corresponding values of \(\tilde{\pi}_{(0,0)}\) of the three models are:
+
+\begin{align}
+    M = 1: \tilde{\pi}_{(0,0)} &= \mu^4 + \mu^3 \lambda_A = \mu^3 (\mu + \lambda^A) \label{eq:rows-conjecture-1}\\
+    M = 2: \tilde{\pi}_{(0,0)} &= \mu^6 + 2\mu^5 \lambda_A + \mu^4 (\lambda^A)^2 = \mu^4(\mu^2 + 2\mu \lambda_A + (\lambda^A)^2) = \mu^4 (\mu + \lambda^A) ^ 2 \label{eq:rows-conjecture-2}\\
+    M = 3: \tilde{\pi}_{(0,0)} &= \mu^8 + 3 \mu^7 \lambda^A + 3 \mu^6 (\lambda^A)^2 + \mu^5(\lambda^A)^3 \nonumber \\
+    &= \mu^5 (\mu^3 + 3 \mu ^2 \lambda^A + 3 \mu (\lambda^A)^2 + (\lambda^A)^3) \nonumber \\
+    &= \mu^5 (\mu + \lambda^A) ^ 3 \label{eq:rows-conjecture-3}
+\end{align}
+
+It can be observed from equations (\ref{eq:rows-conjecture-1}),( \ref{eq:rows-conjecture-2}) and (\ref{eq:rows-conjecture-3}), that there is a noticeable relationship between them. 
+Thus, a generalised formula for the value of \(\tilde{\pi}_{(0,0)}\) when \(C=1, T=1 \text{ and } N=1\) is given by:
+
+\begin{equation}\label{eq:rows-conjecture-general}
+    \tilde{\pi}_{(0,0)} = \mu^{(N+M)} (\mu + \lambda^A)^M
+\end{equation}
+
+It is important to note here that the above property holds when the system capacity is greater than one as well (\(N \geq 1\)). 
+For instance let us consider a Markov model with \(C\) number of servers, a threshold of \(T\), a system capacity of \(N\) and a parking capacity of \(M\). 
+The equivalent values of \(\tilde{\pi}_{(0,0)}\) can be expressed in terms of an unknown function \(k(C,T,N)\) as:
+
+\begin{equation}
+    \tilde{\pi}_{(0,0)} = \mu^{(N+M)} (k(C,T,N))^M
+\end{equation}
+
+
+\subsubsection{Matrix-tree theorem for directed graphs (Kirchhoff's theorem) \cite{matrix-tree-theorem}: }
```

Comment:
This comes out of nowhere.

Reply:
This is the theorem that I didn't use. It just helped get some insight. I basically "left" it there to so it could be used at some point when we come back to this